gradle/bootstrap-setup.gradle

```diff
@@ -1,3 +1,5 @@
+apply plugin: "org.gradle.java-base"
```

Comment:
Normally you always have the Java plugin in an Xtend project, but we're resolving the bootstrap dependencies in the root project (which is unusual, but necessary for our use case), so we need to add at least the java-base plugin here, so BOM support gets activated.